chap-cheri-riscv.tex

```diff
@@ -229,7 +229,7 @@ \subsection{Unique Architectural Features}
   to be interpreted as expecting capability rather than integer operands
   (reducing opcode footprint while maintaining intentionality).
 \item In the compressed RISC-V encoding, the capability encoding mode allows
-  existing load, store, and jump opcodes to be interpreted as expecting
+  existing load, store, jump, and stack addressing opcodes to be interpreted as expecting
```

Comment:
I don't know that "stack addressing" is the best generic name for C.CINCOFFSET*, but I preferred it over my earlier alternative of "stack adjustment" since C.CINCOFFSET16CSPN doesn't actually adjust CSP.